src/test/java/com/github/mc1arke/sonarqube/plugin/classloader/ClassReferenceElevatedClassLoaderFactoryTest.java

```diff
@@ -45,6 +46,8 @@
 /**
  * @author Michael Clarke
  */
+@Ignore("SonarQube has removed bundled plugins from extensions/plugins.\n" +
+    "See https://jira.sonarsource.com/browse/SONAR-13792")
```

Comment:
I'm not happy with these tests being ignored:
1. The classloader functionality is key to how the plugin works, so needs to have something that proves it works during any changes
2. Marking a test as ignored is likely to lead to it never being maintained; if the test isn't needed then we should delete it

I suspect that these tests can continue to operate if the target class is modified, the build classpath is updated to remove the new extensions directory, and the classloader initialisation in the test updated to reference the new extensions directory.

Reply:
The problem here is that SonarQube doesn't ship any extension in the release zip anymore. They choose to embed the extensions that were there in the previous versions, so the extensions directory is now empty.
What we can do, if you want to keep the tests, is make Gradle download a specific extension jar into the extensions directory only for tests. What do you think?